api/src/main/java/io/opentelemetry/common/Labels.java

```diff
@@ -102,6 +102,10 @@ public static Labels of(
         key5, value5);
   }
 
+  public static Labels of(String[] keyValueLabelPairs) {
+    return sortAndFilterToLabels((Object[]) keyValueLabelPairs);
+  }
+
   private static Labels sortAndFilterToLabels(Object... data) {
     return new AutoValue_Labels_ArrayBackedLabels(sortAndFilter(data));
   }
```

Comment:
One of this 2 can be removed.

Reply:
yeah, it's a good point. If we move the metric APIs to use Labels explicitly, rather than varargs strings, I think we can get rid of the one I just added, altogether. I'll clean this up for this PR, though, since there no reason to have the extra method there for now.

Reply:
We can do in a follow-up. Up to you to decide

Reply:
yeah, let's do it in a follow-up. It's actually a bunch messier than I first suspected, due to some fun java reasons. :)